.env

```diff
@@ -8,10 +8,10 @@ APP_DESCRIPTION=Visualization, Exploration, and Data Analysis (VEDA)
 APP_CONTACT_EMAIL=email@example.org
 
 # Endpoint for the Tiler server. No trailing slash.
-API_RASTER_ENDPOINT='https://staging-raster.delta-backend.com'
+API_RASTER_ENDPOINT='https://dev-raster.delta-backend.com'
```

Comment:
Same with this. Can you revert this to staging?

Reply:
Should I revert the endpoint for STAC server to staging as well?

Reply:
Done. Can you check commit 4a638eb?